package.json

```diff
@@ -90,6 +90,7 @@
     "gitbook-plugin-github": "^2.0.0",
     "in-publish": "^2.0.0",
     "istanbul": "^1.0.0-alpha.2",
+    "istanbul-api": "~1.2",
```

Comment:
Where is this being used?

Reply:
it's a dep of istanbul-api; see https://github.com/istanbuljs/istanbuljs/issues/216